pandas/core/arrays/string_arrow.py

```diff
@@ -349,6 +349,15 @@ def __getitem__(self, item: Any) -> Any:
                     "Only integers, slices and integer or "
                     "boolean arrays are valid indices."
                 )
+        elif isinstance(item, tuple):
+            # possibly unpack arr[:, n] to arr[n]
```

Comment:
Why is this needed for arrow string array? That's invalid indexing for a 1D array, and should raise an IndexError (as numpy does, or as we do for eg StringArray)

Reply:
typo, should be `arr[..., n]`, will update

Reply:
Even for `arr[..., n]`, why is it needed specifically here for Arrow string array? That also doesn't work for StringArray

Reply:
got a test failure without this; will add a base extension test to require it for all EAs

Reply:
Do you remember which test?

Reply:
(BTW, I am not sure we should require this behaviour from EAs, as we can't be consistent with numpy in returning a 0-dim array)

Reply:
> Do you remember which test?

test_groupby_extension_apply * 8